.circleci/continue_config.yml

```diff
@@ -57,8 +57,8 @@ executors:
       - image: prestocpp/prestocpp-avx-centos:root-20230613
     resource_class: 2xlarge
     environment:
-      MAVEN_OPTS: "-Xmx4G -XX:+ExitOnOutOfMemoryError"
-      MAVEN_INSTALL_OPTS: "-Xmx2G -XX:+ExitOnOutOfMemoryError"
+      MAVEN_OPTS: "-Xmx4G -XX:+ExitOnOutOfMemoryError -XX:CompressedClassSpaceSize=3g"
```

Comment:
Small recommendation: I see that we are using machine size as 2xlarge (which is 32 GB memory) we can either increase Xmx for both `MAVEN_OPTS` and `MAVEN_FINAL_OPTS` or if we don't need such large machines we can plan to shift to smaller resource class probably to save costs. 

If there are certain suites that require larger machine size, we can also segregate different suites to use different executors to optimize costs.

Reply:
Thanks for the suggestion @imjalpreet. @majetideepak could you please confirm if I can increase Xmx to 8G in `MAVEN_OPTS` and `MAVEN_FINAL_OPTS` in this PR?

Reply:
We have to remember that Prestissimo workers also run on the same node and they require memory. I would prefer to limit the JVM memory to only the required since we know the memory is held up by the JVM until GC runs which is not in our control.